.github/workflows/stale.yaml

```diff
@@ -0,0 +1,46 @@
+name: Mark and Close Stale Issues and PRs
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # runs every day at midnight UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Mark and close stale issues and PRs
+        uses: actions/stale@v9
+        with:
+          stale-issue-label: 'lifecycle/stale'
+          exempt-issue-labels: 'reminder'
```

Comment:
Do we need to also add `exempt-pr-labels`?

Reply:
sure.

Reply:
what should be the values?

Reply:
cc @Monokaix

Reply:
I have no advice on this, let @Monokaix  decide